src/foam/mlang/mlang.js

```diff
@@ -3812,6 +3812,7 @@ foam.CLASS({
     {
       name: 'f',
       code: function(obj) {
+        if ( ! obj ) return null;
```

Comment:
This function is expected to return a boolean, so it would make more sense to 'false'.
Also, can you track down and fix where this is being passed a null in the first place?

Reply:
oh, right. Okay, I will check it.